plenary/plenary-21.yaml

```diff
@@ -0,0 +1,287 @@
+        message: notes the on-going work (see also Resolution 230 and Action 15 above, both open) managed by Mr. L. Visser (NEN/NL): Thanks a lot, Louis !
+      - type: confirms
+        date_effective: 2003-09-12
+        message: confirms its availability to launch a ballot on this draft revision, when available
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "ISO11180:1993 postal addressing (see also doc. 154n448)"
+    identifier: 244
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: recalls
+        date_effective: 2003-09-12
+        message: recalling that trade in goods and services need to identify, in an open environment allowing interoperability, the parties and the location of the commodity, most of which parties and locations having a name and address, processable by all parties and all modes of transport and usable across all syntaxes
+      - type: recalls
+        date_effective: 2003-09-12
+        message: recalling that ISO11180:1993 had been launched by UPU
+      - type: recalls
+        date_effective: 2003-09-12
+        message: recalling core inconsistencies between two ISO/TC154 standards: ISO6422 Layout Key (e.g. 5x35 car.) and ISO11180 (e.g. 6x30 car.)
+      - type: notes
+        date_effective: 2003-09-12
+        message: noting that ISO11180 has not been widely implemented by ISO members
+      - type: notes
+        date_effective: 2003-09-12
+        message: noting that ISO11180 is no longer referenced by UPU (see e.g. the new UPU S42 standard)
+      - type: resolves
+        date_effective: 2003-09-12
+        message: resolves to withdraw ISO11180:1993
+      - type: resolves
+        date_effective: 2003-09-12
+        message: resolves to investigate the possibility of launching an ISO New Work Item (NWI) on name and address (follow-up: see action 19)
+
+    # SECTION 19
+    # 1. The ISO/TC154 chair is tasked with the preparatory steps (draft job description and membership) for this possible ISO New Work Item (NWI) on name and address
+    # 2. draft job description: [open]
+    # 3. draft membership: this possible NWI could involve the following ISO/TC154 P(active)-members, at least 5 (identified contact persons):
+    # - BE Mr. J.-L. Champion
+    # - CH Mr. F. Vuilleumier
+    # - DE Mr. M. Dill
+    # - GB Ms. S.L. Probert
+    # - JP Mr. K. Wakaizumi
+    # - KR Mr. Y.-G. Jung
+    # - US Ms. M.K. Blantz
+    # in co-operation with the following ISO/TC154 L(iaison)-members:
+    # - UPU (standard S42),
+    # - CEN/TC331 (standard CEN14142)
+    # - ISO/TC215 as well as other interested TC’s
+    # - ISO/IEC JTC1/SC32 as well as other interested SC’s
+    # - UN/CEFACT Forum: at least TBG, possibly ATG
+    # - UNECE UNeDocs Project
+    # - OASIS Management and UBL TC + CIQ TC
+    # - OAGI
+
```

Comment:
This resolution contains a `section 19` (commented above) that I completely ignored as I don't see how to set it properly.

Source:
![capture11](https://user-images.githubusercontent.com/33627611/71759882-db6deb80-2e8a-11ea-8817-68580173e248.JPG)
![capture12](https://user-images.githubusercontent.com/33627611/71759883-df017280-2e8a-11ea-8341-af1774c03740.JPG)

Reply:
The points with "noting", "recalling" are considerations. This will work better:

```yaml
    considerations:
      - type: recalling
        date_effective: 2003-09-12
        message: recalling that trade in goods and services need to identify, in an open environment allowing interoperability, the parties and the location of the commodity, most of which parties and locations having a name and address, processable by all parties and all modes of transport and usable across all syntaxes
      - type: recalling
        date_effective: 2003-09-12
        message: recalling that ISO 11180:1993 had been launched by UPU
      - type: recalling
        date_effective: 2003-09-12
        message: |
          recalling core inconsistencies between two ISO/TC154 standards: ISO 6422 Layout Key (e.g. 5x35 car.) and ISO 11180 (e.g. 6x30 car.)
      - type: noting
        date_effective: 2003-09-12
        message: noting that ISO 11180 has not been widely implemented by ISO members
      - type: noting
        date_effective: 2003-09-12
        message: noting that ISO 11180 is no longer referenced by UPU (see e.g. the new UPU S42 standard)

    actions:
      - type: withdraws
        date_effective: 2003-09-12
        message: resolves to withdraw ISO 11180:1993

      - type: investigates
        date_effective: 2003-09-12
        message: |
          resolves to investigate the possibility of launching an ISO New
          Work Item (NWI) on name and address (follow-up: see action 19)

          Action 19

          1. The ISO/TC154 chair is tasked with the preparatory steps (draft
            job description and membership) for this possible ISO New Work Item
            (NWI) on name and address
          2. draft job description: [open]

          3. draft membership: this possible NWI could involve the following
            ISO/TC154 P(active)-members, at least 5 (identified contact
            persons):

            - BE Mr. J.-L. Champion
            - CH Mr. F. Vuilleumier
            - DE Mr. M. Dill
            - GB Ms. S.L. Probert
            - JP Mr. K. Wakaizumi
            - KR Mr. Y.-G. Jung
            - US Ms. M.K. Blantz

          in co-operation with the following ISO/TC154 L(iaison)-members:
            - UPU (standard S42),
            - CEN/TC331 (standard CEN14142)
            - ISO/TC215 as well as other interested TC’s
            - ISO/IEC JTC1/SC32 as well as other interested SC’s
            - UN/CEFACT Forum: at least TBG, possibly ATG
            - UNECE UNeDocs Project
            - OASIS Management and UBL TC + CIQ TC
            - OAGI
```